test/e2e-common.sh

```diff
@@ -293,36 +322,36 @@ function test_setup() {
 
 # Tear down resources used in the eventing tests.
 function test_teardown() {
-  uninstall_test_resources
+  true
```

Comment:
hm, Is this not done in this section?

Reply:
I didn't understand why imc-channel should be installed/uninstalled separately, so I moved it together with other components.
https://github.com/knative/eventing/pull/4341/files#diff-fbb940d8ac3d40d6c801e8a40575b2a3cd6d6e3f1d2d8f8122fbc2c438a6dccfR347

Reply:
ah, because imc is not always installed.

Reply:
ok